include/mxnet/c_predict_api.h

```diff
@@ -56,6 +56,16 @@ typedef void (*PredMonitorCallback)(const char*,
                                     NDArrayHandle,
                                     void*);
 
+/*! \brief enum of NDArray dtypes */
+enum DType {
```

Comment:
can we reuse this? https://github.com/apache/incubator-mxnet/blob/master/3rdparty/mshadow/mshadow/base.h#L352
Keeps the code clean by avoiding multiple copies of different Enums

Reply:
no, this is part of the c_predict_api.h header file that does not import any other MX includes. its standalone.